packages/2d/src/utils/patchSignal.ts

```diff
@@ -27,5 +27,12 @@ export function patchSignal<TSetterValue, TValue extends TSetterValue>(
     if (tweener) {
       signal.doTween = tweener.bind(owner);
     }
+    const computer = owner?.[`compute${capitalize(name)}`];
+    if (computer) {
+      signal.compute = computer.bind(owner);
+      if (signal['initial'] === undefined) {
```

Comment:
I think we can just make `initial` public since there's already a setter for it and now we're also getting it.